packages/financial-templates-lib/src/helpers/GasEstimator.ts

```diff
@@ -61,7 +61,7 @@ export const MAPPING_BY_NETWORK: GasEstimatorMapping = {
   },
   10: { defaultFastPriceGwei: 1, type: NetworkType.Legacy },
   137: {
-    url: "https://api.polygonscan.com/api?module=gastracker&action=gasoracle",
+    url: "https://gasstation.polygon.technology/v2",
```

Comment:
`_extractFastGasPrice` still references `polygonscan.com` in URL:
```
    if (url.includes("etherscan.io") || url.includes("polygonscan.com")) {
```
Might need to remove it there so it only references `etherscan.io` in that branching logic